src/services/styleurl/styleurl.helper.ts

```diff
@@ -0,0 +1,72 @@
+const VECTORTILES_URL = import.meta.env.VITE_VECTORTILES_URL
+const MVTSTYLES_PATH_GET = import.meta.env.VITE_MVTSTYLES_PATH_GET
+const MVTSTYLES_PATH_UPLOAD = import.meta.env.VITE_MVTSTYLES_PATH_UPLOAD
+const MVTSTYLES_PATH_DELETE = import.meta.env.VITE_MVTSTYLES_PATH_DELETE
+
+class StyleUrlHelper {
+  private _vectortilesUrl = VECTORTILES_URL
```

Comment:
The `_...` syntax reminds me of v3 :-)

Reply:
I made a copy of: https://github.com/Geoportail-Luxembourg/luxembourg-geoportail/pull/95
and did not think much